modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectEntry/ModalDeleteObjectEntry.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayModal from '@clayui/modal';
+import {fetch, openToast} from 'frontend-js-web';
+import React from 'react';
+
+interface ItemData {
+	actions: ItemDataDelete;
+	id: number;
+}
+
+interface ItemDataDeleteProps {
+	href: string;
+	method: string;
+}
+
+interface ModalDeleteObjectEntry {
+	closeModal: voidReturn;
+	itemData: ItemData;
+	loadData: voidReturn;
+}
+
+type ItemDataDelete = {
```

Comment:
@Artartur I think it's not necessary to create this type. You can declare it inline, at the moment of the usage.

Reply:
Done!